### PR TITLE
Remove 32-bit linux wheel builds

### PR DESCRIPTION
**Pull request by brisvag**
PR with no changes to see if CI can build (to check for #2285).

### CODE REVIEW

**Comment by brisvag** [comment]
@djhoese the issue does not seem to be related to the changes in #2285 at least...

**Comment by djhoese** [comment]
Yeah. I figured. Darn though. Let's see, what can we do to test this. It looks like the tests hang in the 32-bit (i686) builds of the Python 3.8 wheel. As a test, what if you added `cp38-manylinux_i686` to https://github.com/vispy/vispy/blob/main/.github/workflows/wheels.yml#L25 in this PR and we see if it passes. If it hangs again but on Python 3.9 then we may have to stop producing 32-bit wheels all together.

We could also play with versions of dependencies but that's a pretty deep rabbit hole to dive in to. We could try running the wheel building locally, but I'm not sure how comparable that is to the CI environment. I suppose with docker containers and 32-bit emulation we don't need an actual 32-bit system?

@kmuehlbauer @larsoner any other ideas?

**Comment by djhoese** [comment]
Still hanging. Want to add `cp39`?

**Comment by djhoese** [comment]
Ok so 32-bit builds are the issue. Do we stop producing 32-bit builds or spend more time trying to figure this out?

**Comment by brisvag** [comment]
It seems that [numpy has dropped linux 32 bit wheels](https://github.com/numpy/numpy/releases):

```
The Python versions supported in this release are 3.8-3.10, Python 3.7
has been dropped. Note that 32 bit wheels are only provided for Python
3.8 and 3.9 on Windows, all other wheels are 64 bits on account of
Ubuntu, Fedora, and other Linux distributions dropping 32 bit support.
All 64 bit wheels are also linked with 64 bit integer OpenBLAS, which should fix
the occasional problems encountered by folks using truly huge arrays.
```

So I guess we should just drop it.

**Comment by djhoese** [comment]
Yes, that's why we had the ignore for Python 3.10 32-bit wheels, but older versions of Python were still working. @vispy/core any objections to stopping all 32-bit builds (since we can't seem to get them to complete).

**Comment by brisvag** [comment]
But numpy only dropped support for py37. The latest release (numpy-1.22.1) has wheels for py38, py39 and py10. So maybe the issue is that CI is trying to find the wheel for the latest release but can't find it for the 32bit architecture? Not sure how this kind of dependency wheel search is handled.

**Comment by djhoese** [comment]
Keep in mind both python version and architecture (32 versus 64-bit). The wheel building is failing/hanging during testing, not during installation of dependencies. It has nothing to do with installation of numpy. The build hangs on 32-bit Python 3.8 and 32-bit Python 3.9 and there are numpy wheels for both.

**Comment by larsoner** [comment]
I would just drop 32-bit linux

File: .github/workflows/wheels.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
     - name: Build wheels
       uses: pypa/cibuildwheel@v2.1.1
       env:
-        CIBW_SKIP: "cp27-* cp34-* cp35-* cp36-* pp* *-win32 cp310-manylinux_i686"
+        CIBW_SKIP: "cp27-* cp34-* cp35-* cp36-* cp38-manylinux_i686 cp39-manylinux_i686 pp* *-win32 cp310-manylinux_i686"
```
Comment (djhoese) [inline .github/workflows/wheels.yml:25]: Not sure this wildcard will work, but:

```suggestion
        CIBW_SKIP: "cp27-* cp34-* cp35-* cp36-* pp* *-win32 *-manylinux_i686"
```

**Comment by djhoese** [comment]
Thanks @brisvag!

**Pull request merged**